docs/source/install.rst

```diff
@@ -33,12 +33,12 @@ want to consider one of the alternate worker types.
 
 ::
 
-    $ pip install greenlet  # Required for both
-    $ pip install eventlet  # For eventlet workers
-    $ pip install gevent    # For gevent workers
+    $ pip install gunicorn[eventlet]  # For eventlet workers
+    $ pip install gunicorn[gevent]    # For gevent workers
```

Comment:
i’m not sure about this changes. some people are installing gunicorn with both workers support and not always at the same time. I would suggest to keep it like this and maybe add a new line.